### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by tiagocardosoweb**
Add version into App Bar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <meta charset='utf-8'>
   <meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no'>
   <meta http-equiv='x-ua-compatible' content='ie=edge'>
-  <title>Pokémon GO - LocationSpoof</title>
+  <title>Pokémon GO - LocationSpoof v1.1.0</title>
```
Comment (d3viant0ne) [inline index.html]: @tiagocardosoweb - You should really pull the version from the package.json so this doesn't need to be updated manually every time a new version is released.

**Comment by Xjkstar** [comment]
can add a feature：
Live map visualization of all pokemon in your area with options to also show gyms, filter pokemon, localization, and pokestops. 
like this：
https://github.com/AHAAAAAAA/PokemonGo-Map

**Comment by ataetgi** [comment]
is it possible mixing with https://github.com/AHAAAAAAA/PokemonGo-Map this?

**Comment by d3viant0ne** [comment]
@Xjkstar @ataetgi - A pull request for the index.html is not the proper place for feature requests, particularly when completely unrelated to the code change.

If you feel strongly about the feature, open an issue assuming one doesn't exist and it can be discussed there.

**Pull request closed without merging**